Nano.Net/RpcClient.cs

```diff
@@ -88,8 +88,17 @@ public async Task<AccountInfoResponse> AccountInfoAsync(string address, bool rep
         /// <remarks>
         /// WARNING: This command is usually disabled on public nodes. You need to use your own node.
         /// </remarks>
-        public async Task<WorkGenerateResponse> WorkGenerateAsync(string hash)
+        public async Task<WorkGenerateResponse> WorkGenerateAsync(string hash, string difficulty = "")
```

Comment:
The default difficulty argument should be null and not an empty string to maintain consistency across the library.

Reply:
Please check 981ddd0.